apps/src/templates/teacherNavigation/TeacherNavigationPaths.tsx

```diff
@@ -25,45 +30,84 @@ export const LABELED_TEACHER_NAVIGATION_PATHS = {
     url: TEACHER_NAVIGATION_PATHS.progress,
     absoluteUrl: getAbsolutePath(TEACHER_NAVIGATION_PATHS.progress),
     label: i18n.progress(),
+    icon: 'chart-line',
   },
   textResponses: {
     url: TEACHER_NAVIGATION_PATHS.textResponses,
     absoluteUrl: getAbsolutePath(TEACHER_NAVIGATION_PATHS.textResponses),
     label: i18n.teacherTabStatsTextResponses(),
+    icon: 'pen-line',
   },
   assessments: {
     url: TEACHER_NAVIGATION_PATHS.assessments,
     absoluteUrl: getAbsolutePath(TEACHER_NAVIGATION_PATHS.assessments),
     label: i18n.assessments(),
+    icon: 'star',
   },
   projects: {
     url: TEACHER_NAVIGATION_PATHS.projects,
     absoluteUrl: getAbsolutePath(TEACHER_NAVIGATION_PATHS.projects),
     label: i18n.studentProjects(),
+    icon: 'code',
   },
   stats: {
     url: TEACHER_NAVIGATION_PATHS.stats,
     absoluteUrl: getAbsolutePath(TEACHER_NAVIGATION_PATHS.stats),
     label: i18n.teacherTabStats(),
+    icon: 'chart-simple',
   },
   manageStudents: {
     url: TEACHER_NAVIGATION_PATHS.manageStudents,
     absoluteUrl: getAbsolutePath(TEACHER_NAVIGATION_PATHS.manageStudents),
     label: i18n.roster(),
+    icon: 'users',
   },
   loginInfo: {
     url: TEACHER_NAVIGATION_PATHS.loginInfo,
     absoluteUrl: getAbsolutePath(TEACHER_NAVIGATION_PATHS.loginInfo),
     label: null,
+    icon: 'users',
   },
   standardsReport: {
     url: TEACHER_NAVIGATION_PATHS.standardsReport,
     absoluteUrl: getAbsolutePath(TEACHER_NAVIGATION_PATHS.standardsReport),
     label: null,
+    icon: 'users',
   },
   aiTutorChatMessages: {
     url: TEACHER_NAVIGATION_PATHS.aiTutorChatMessages,
     absoluteUrl: getAbsolutePath(TEACHER_NAVIGATION_PATHS.aiTutorChatMessages),
     label: null,
+    icon: 'users',
+  },
+  lessonMaterials: {
+    url: TEACHER_NAVIGATION_PATHS.lessonMaterials,
+    absoluteUrl: getAbsolutePath(TEACHER_NAVIGATION_PATHS.lessonMaterials),
+    label: i18n.lessonMaterials(),
+    icon: 'folder-open',
+  },
+  calendar: {
+    url: TEACHER_NAVIGATION_PATHS.calendar,
+    absoluteUrl: getAbsolutePath(TEACHER_NAVIGATION_PATHS.calendar),
+    label: i18n.calendar(),
+    icon: 'calendar',
+  },
+  courseOverview: {
+    url: TEACHER_NAVIGATION_PATHS.courseOverview,
+    absoluteUrl: getAbsolutePath(TEACHER_NAVIGATION_PATHS.courseOverview),
+    label: i18n.course(),
+    icon: 'desktop',
+  },
+  unitOverview: {
+    url: TEACHER_NAVIGATION_PATHS.unitOverview,
+    absoluteUrl: getAbsolutePath(TEACHER_NAVIGATION_PATHS.unitOverview),
+    label: i18n.unit(),
+    icon: 'users',
```

Comment:
This is also a placeholder icon.  It is unclear to me where the `unitOverview` is going to be shared... I have this as a question for Bryan/Molly in check in.